spec/controllers/control_panel/control_panel_spec.rb

```diff
@@ -1,20 +1,174 @@
 require 'rails_helper'
 
-RSpec.describe ControlPanel::StaticPagesController do
+RSpec.describe ControlPanel::BoardsController do
```

Comment:
Clean up this file. Split it on two contexts.
```
'with logged in user'
'without logged in user'
```
then you can remove the duplication ( sing_in :user )